src/component/Header/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {FC} from 'react';
+import './Header.scss';
+import './HeaderResponsive.scss';
+import {Logo} from "../Logo/Logo";
+import {UserProfile} from "../UserProfile/UserProfile";
+import { NavLink } from 'react-router-dom';
+
+export const Header: FC = () => {
+    return (
+        <>
+            <div id="main-header" className="main-header w-100 main-bg">

```

Comment:
we are already using HTML5 so we should use header tag with header id for main header.